src/tpg_tcp.c

```diff
@@ -275,6 +275,14 @@ int tcp_close_connection(tcp_control_block_t *tcb, uint32_t flags)
      * For silent close all we do is cleanup the tcb
      */
     if ((flags & TCG_SILENT_CLOSE) != 0) {
+        /* don't send a RST when closing after a RST was received */
+        if (!tcb->tcb_rst_rcvd) {
+            //tpg_port_statistics_t *stats =
```

Comment:
Can you please remove the commented code?

Reply:
Done.